_i18n/en/resources/user-guides/join-monero-matrix.md

```diff
@@ -1,34 +1,37 @@
 {% include disclaimer.html translated="no" translationOutdated="no" %}
 
-Monero developers primarily communicate via IRC and Matrix channels. You can join IRC/Libera and Matrix rooms using Matrix. An open source network for secure, decentralized communication.
+Monero contributors primarily communicate via IRC and Matrix channels. You can join IRC/Libera and Matrix rooms using Matrix. An open source network for secure, decentralized communication.
 
-In order to access the Matrix protocol you will need to use a Matrix client. A popular client is Element. Element is a free, open-source program that can be accessed via web, desktop, and mobile.
+In order to access the Matrix protocol you will need to use a Matrix client. In this tutorial we will use Element as a client. Element is a free, open-source program that can be accessed via web, desktop, and mobile.
 
 ### Getting Element
 
 Go to [element.io](https://element.io/get-started#download) to download the latest version of the Element app on your computer. Alternatively you can use the Element [web app](https://app.element.io). There are Android apps available on [F-Droid](https://f-droid.org/packages/im.vector.app/) or [Google Play](https://play.google.com/store/apps/details?id=im.vector.app). As well as an app for [iPhone](https://apps.apple.com/app/vector/id1083446067). Once you have accessed the client of your choice create an account on our host server.
 
 ### Setting the custom host server
 
-While matrix is a federated protocol and you could use any host server for your account (including the default matrix.org server). We recommend changing your server to the one hosted by the Monero core team to support decentralization. If you do choose this option remember to set your custom server when logging in on a new device.
+While matrix is a federated protocol and you could use any host server for your account (including the default matrix.org server). We recommend changing your server to the one hosted by the Monero community members to support decentralization. If you do choose this option remember to set your custom server when logging in on a new device.
```

Comment:
only nit pick here is monero social is hosted by cypherstack. the general fund pays for this. its equivalent to saying that getmonero is run by community members

Reply:
ah yes, correcting my prev comment, cake sponsor the server, cypherstack provide 1 back end admin (https://www.getmonero.org/community/sponsorships/) aka "the Monero community members". 

docs.getmonero.org on the other hand, definitely hosted by "the monero community members"

Reply:
```suggestion
While matrix is a federated protocol and you could use any host server for your account (including the default matrix.org server). We recommend changing your server to the one hosted / maintained by the Monero project [sponsors]({{ site.baseurl }}/community/sponsorships/. If you do choose this option remember to set your custom server when logging in on a new device.
```
thoughts?

Reply:
LGTM, thoughts @HardenedSteel?